src/live.js

```diff
@@ -0,0 +1,32 @@
+export function saveLocal(list) {
+  window.localStorage.setItem('localTasks', JSON.stringify(list));
+}
+
+export function status(elem, list) {
+  list.forEach((task) => {
+    if (task === elem) {
+      task.isCompleted = !task.isCompleted;
+    }
+  });
+  saveLocal(list);
+}
+
+export function add(list) {
+  list.push({ description: document.querySelector('#newTask').value, isCompleted: false, index: list.length });
+  document.querySelector('#newTask').value = '';
+  saveLocal(list);
+}
+
+export function updateIndex(list) {
+  let i = 0;
+  list.forEach((elem) => {
```

Comment:
As mentioned in a previous comment, kindly use destructuring syntax with the objects used as arguments to an array. This helps to keep your code dry and also avoids the importation of object properties that you might not use in the function. 🧷